lib/warnings.pm

```diff
@@ -580,6 +580,9 @@ warnings - Perl pragma to control optional warnings
     use warnings;
     no warnings;
 
+    # Standard warnings are enabled by use v5.35 or above
+    use v5.35;
```

Comment:
Possibly we should ignore 5.35 and make it 5.36 for documentation purposes

Reply:
I'd suggest leave it at 5.35 for now, and try to remember to change this to 5.36 on the runup to release